images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -0,0 +1,19 @@
+################################################################################
+##  File:  Install-Haskell.ps1
+##  Desc:  Install Haskell for Windows
+################################################################################
+
+# Get 3 latest versions of GHC
+[Version[]] $ChocoVersionsOutput = & choco search ghc --allversions --limit-output | Where-Object { $_.StartsWith("ghc|") } | ForEach-Object { $_.TrimStart("ghc|") }
+$MajorMinorGroups = $ChocoVersionsOutput | Sort-Object -Descending | Group-Object { $_.ToString(2) } | Select-Object -First 3
+$VersionsList = $MajorMinorGroups | ForEach-Object { $_.Group | Select-Object -First 1 }
+
+# The latest version will be installed as a default
```

Comment:
Am I right that the latest version will be 8.6.5?

```
Major  Minor  Build  Revision
-----  -----  -----  --------
8      10     1      -1
8      8      3      -1
8      6      5      -1
```

Reply:
I think it is better to do `Sort-Object` without `Descending` and take latest 3 elements

Reply:
Yes, last installed version is being considered by system as a default version.

Reply:
Fixed the order. Now  8.10.1 is last in installation list and will be considered as a default version